translate/public/locale/en-US/translate.ftl

```diff
@@ -682,11 +682,15 @@ search-FiltersPanel--apply-filters =
 
 ## Search Options Panel
 
-search-SearchPanel--option-name-identifiers = Search in string identifiers
+search-SearchPanel--option-name-search_match_case = Match case
+search-SearchPanel--option-name-search_match_whole_word = Match whole word
+search-SearchPanel--option-name-search_identifiers = Search in string identifiers
+search-SearchPanel--option-name-search_rejected_translations = Include rejected translations
+search-SearchPanel--option-name-search_exclude_source_strings = Exclude source strings
```

Comment:
It would be nice to be consistent with the rest of the file and the README and use the kebab-case:
https://github.com/mozilla/pontoon/blob/0788ed9de8b71b3ab9cb1fb76bb40fe0fe9fa5e0/translate/README.md#localization